src/Maestro/DependencyUpdateErrorProcessor/DependencyUpdateErrorProcessor.cs

```diff
@@ -0,0 +1,227 @@
+            )
+        {
+            StateManager = stateManager;
+            Logger = logger;
+            Context = context;
+            DependencyUpdateErrorProcessorOptions = dependencyUpdateErrorProcessorOptions.Value;
+        }
+
+        public IReliableStateManager StateManager { get; }
+
+        public ILogger<DependencyUpdateErrorProcessor> Logger { get; }
+
+        public BuildAssetRegistryContext Context { get; }
+
+        public DependencyUpdateErrorProcessorOptions DependencyUpdateErrorProcessorOptions { get; }
+
+        [CronSchedule("0 0 0/1 1/1 * ? *", TimeZones.PST)]
+        public async Task DependencyUpdateErrorProcessing()
+        {
+
+            if (DependencyUpdateErrorProcessorOptions.ConfigurationRefresherdPointUri != null && DependencyUpdateErrorProcessorOptions.DynamicConfigs != null)
+            {
+                DependencyUpdateErrorProcessorOptions.ConfigurationRefresherdPointUri.Refresh().GetAwaiter().GetResult();
+           
+                bool.TryParse(DependencyUpdateErrorProcessorOptions.DynamicConfigs["FeatureManagement:DependencyUpdateErrorProcessor"], 
+                    out var autoDependencyUpdateErrorProcessor);
+                if (autoDependencyUpdateErrorProcessor)
+                {
+                    IReliableDictionary<string, DateTime> update =
+                        await StateManager.GetOrAddAsync<IReliableDictionary<string, DateTime>>("update");
+                    try
+                    {
+                        using (ITransaction tx = StateManager.CreateTransaction())
+                        {
+                            var previousTransactionDateTime = await update.GetOrAddAsync(
+                             tx,
+                             "update",
+                             DateTime.Now
+                             );
+                            var updatedTime = await CheckForError(previousTransactionDateTime, DependencyUpdateErrorProcessorOptions.GithubUrl);
+                            await update.TryUpdateAsync(
+                                tx,
+                                "update",
+                                updatedTime.UtcDateTime,
+                                previousTransactionDateTime
+                                );
+                            await tx.CommitAsync();
+                        }
+                    }
+                    catch (Exception ex)
+                    {
+                        Logger.LogError(ex, "Processing update dependency messages");
+                    }
+                }
+            }
+            else
+            {
+                Logger.LogInformation("Dependency Update Error processor is disabled because no App Configuration was available.");
+            }
+        }
+
+        public async Task<DateTimeOffset> CheckForError(DateTimeOffset previousTransactionDateTime , string whereToCreateIssue)
```

Comment:
This should probably be `TryProessErrorsAsync`, and it should return null/false if it doesn't do anything.  There's not a great reason to commit something do the replicated state store if we didn't do anything.

And "CheckForErrors" could probably have a more descriptive name... right now it sound like it's just running some checks, where, in fact, it's doing a lot more than that (and also returning a DateTime... if I call chadsCar.CheckTirePressure, I don't expect it to charge my credit card and then return "next Tuesday"... I'd expect either a boolean of "it's good" or some sort of goodness value returned).